scipy/signal/tests/test_peak_finding.py

```diff
@@ -822,3 +822,36 @@ def test_find_peaks_nopeak(self):
         widths = np.arange(10, 50)
         found_locs = find_peaks_cwt(test_data, widths, min_snr=5, noise_perc=30)
         np.testing.assert_equal(len(found_locs), 0)
+
+    def test_find_peaks_with_different_width_dtypes(self):
+        """
+        Verify that the `width` argument
+        in `find_peaks_cwt` can take numbers
+        and iterables.
```

Comment:
The test cases below test only lists and tuples --- instead of them, a single test case for a float input would be sufficient (arrays are tested in the other tests).
The point of the test here would be only to test the part changed in this PR.